### ISSUE TITLE
Animation not allowing to navigate

### ISSUE THREAD

**Issue by waheedakhtar694** [PedroBern/react-native-collapsible-tab-view#233]
@andreialecu  I have three tabs and while scrolling animation is in progress if i press some button either to change tab or navigate on some other screen it's not allowing me to change tab or navigate until scrolling gets completed, Is there a way to cancel scrolling animation?

**Comment by azwinlam** [comment]
See problem #221

**Comment by waheedakhtar694** [comment]
@azwinlam i am using reanimated `2.2.4` and i am able to scroll but my issue is while scrolling animation in progress it halts the other touchable component.

**Comment by azwinlam** [comment]
Right. We also didn't have this problem until we upgraded some packages. We
also tried downgrading to 2.2.4 with nil help. But I believe both thread is
having the same issues.

**Comment by khlling** [comment]
Probably related to https://github.com/software-mansion/react-native-reanimated/issues/1701
